core/Classes/File.php

```diff
@@ -224,7 +224,7 @@ public static function makeDir(string $dir)
     {
         $dir = str_replace('/', DIRECTORY_SEPARATOR, $dir);
 
-        if ((isWindows() && !preg_match('/^\w:(\\|\/)/', $dir)) || (!isWindows() && substr($dir, 0, 1) != DIRECTORY_SEPARATOR)) {
```

Comment:
Why did you remove the `!`? Now the statement is true when the path starts with C:/ or D:\ I see that I forgott an i to make it case-insensitive

Reply:
Ah, I forgot to include a comment there - for some reason, in a new install, it fails with the ! sign added (in MapController.php, line 53 - I see I included a comment there). Without the sign, it worked fine. I am honestly not sure why it didn't work, but maybe it's just for me.

Reply:
That is because the regex is currently case-sensitive and only checks for c, d not C, D. I need to make it //i